company-box.el

```diff
@@ -839,13 +840,17 @@ It doesn't nothing if a font icon is used."
           (frame (company-box--get-frame (frame-parent)))
           (window (frame-local-getq company-box-window (frame-parent)))
           (char-width (frame-char-width frame))
+          (current-width (* (frame-width frame) char-width))
```

Comment:
```suggestion
          (current-width (frame-inner-width frame))
```

@leodag Can you please use `frame-inner-width` here, it's a faster alternative to get the width.
`frame-width` traverses the list of frame parameters, and that list can be very long on some configuration